docs/installation.md

```diff
@@ -4,24 +4,14 @@
 
 govuk-docker has the following dependencies:
 
-- [brew](https://brew.sh/). (If you don't use a Mac, you'll need to dig into the `govuk-docker setup` command and manually install the things referenced).
+- [brew](https://brew.sh/)
+  - If you don't use a Mac, see the [Brewfile](../Brewfile)
```

Comment:
Homebrew also works on Linux and last time I tried, this Brewfile _does_ work apart from the Docker `cask` (I supressed the urge to include a `brew "docker" if OS.linux?`). But Linux users will have other problems (notably being constrained to the BYOD VPN). So it's best if we don't?

Reply:
Hmm...I'm not sure. I don't think we should make the instructions more complicated, but they should also not be confusing for anyone who is using Linux. This seems like a reasonable compromise, since they are still motivated to ensure Docker is installed.

Reply:
There are other changes to make govuk-docker work for Linux, including dnsmasq and resolver configs. I have a branch somewhere from a few months ago. Everything's changed since then, and I stopped working on it when I realised that the BYOD VPN scuppered my use of it. I'm happy to revisit them if there's a user need, but for now I think there's not until Linux laptops become more usual in GOV.UK (which doesn't seem to be a Thing that we or IT are pushing for).

Reply:
Ah yes, I forgot about that. I've updated the commit/message to remove the Mac caveat, pending a proper set of instructions. Seems reasonable to leave the Docker settings bit, since it's separate and specific to Macs.